### PR TITLE
Accessibility fixed for input

### PR DESCRIPTION
**Pull request by bhardwajamit0890**

### CODE REVIEW

File: config/jest.setup.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
 const { configure } = require('enzyme');
 const Adapter = require('enzyme-adapter-react-16');
 
+const lodash = require.requireActual('lodash');
+
 configure({ adapter: new Adapter() });
+lodash.uniqueId = () => {
+  return 1;
```
Comment (vinodloha) [inline config/jest.setup.js]: This is not good enough mock.

File: lib/components/molecules/Form/FieldInput/FieldInput.js [hunk 2]
```diff
@@ -60,13 +66,21 @@ const FieldInput = ({
           );
         }}
       </Field>
+      {validationInstructions ? (
+        <Para id={`${inputID}-instruction`} className="hide-default-sample">
```
Comment (vinodloha) [inline lib/components/molecules/Form/FieldInput/FieldInput.js]: className="hide-default-sample" class is just for storybook. Let this text be visible

File: package.json [hunk 3]
```diff
@@ -133,6 +133,7 @@
     "core-js": "^2.6.5",
     "flexboxgrid": "^6.3.1",
     "formik": "^2.0.6",
+    "lodash": "^4.17.15",
```
Comment (manishekhawat) [inline package.json:136]: if we are introducing lodash only for this utility of uniqueId across the application, then lets write our own logic to generate uniqueId or just copy that particular utility method. @vinodloha your thoughts?

Comment (ankurkakroo2) [reply package.json:136]: @bhardwajamit0890  Lodash has modularised their code a lot. 
Just use: https://www.npmjs.com/package/lodash.uniqueid
I know what you have done will also result in a tree-shaken code, but still ;)

**Review by vinodloha — APPROVED** [review]

**Pull request merged**